docs/fsharp/style-guide/formatting.md

```diff
@@ -301,6 +301,15 @@ Inline comments should capitalize the first letter.
 let f x = x + 1 // Increment by one.
 ```
 
+Formatting interpolated strings
+-------------------------------
+
+Interpolated strings (and plain strings literals) can just be left on a single line, regardless of how long the line is.
```

Comment:
How can `plain strings literals` be written any differently?

Reply:
I guess that depends on why you want it written differently. I gave it a try now.